demos/projects/ESPRESSIF/esp32/components/sample-azure-iot/mbedtls_error.h

```diff
@@ -0,0 +1,67 @@
+/*
+ * FreeRTOS V202104.00
```

Comment:
you do not need this file and corresponding c file, as they are not used.

Reply:
Removed.